src/cmd/terraform.go

```diff
@@ -310,30 +310,49 @@ func exportServices(c *opslevel.Client, shell *os.File, directory string) {
 func exportTeams(c *opslevel.Client, config *os.File, shell *os.File) {
 	shell.WriteString("# Teams\n")
 
-	teamConfig := `resource "opslevel_team" "%s" {
-  name = "%s"
-  manager_email = "%s"
-  parent = "%s"
-  %s
-  %s
+	teamConfig := `resource "opslevel_team" "%s" {%s
 }
+
 `
 	resp, err := c.ListTeams(nil)
 	teams := resp.Nodes
 	cobra.CheckErr(err)
 	for _, team := range teams {
+		teamBody := ""
+
 		aliases := flattenAliases(team.Aliases)
 		if len(aliases) > 0 {
-			aliases = fmt.Sprintf("aliases = [\"%s\"]", aliases)
+			teamBody += fmt.Sprintf("\n  aliases = [\"%s\"]", aliases)
 		}
+		teamBody += fmt.Sprintf("\n  name = \"%s\"", team.Name)
+
+		if len(team.Group.Alias) > 0 {
```

Comment:
Since groups are deprecated do you think this would be better left out?

If we keep it in we should create a follow up ticket to remove this when we officially remove [Group from the Team struct](https://github.com/OpsLevel/opslevel-go/blob/main/team.go#L53)

Reply:
I think it should be kept in up until we remove the field entirely from terraform. Right now the field exists, but is deprecated. There may be some customers who have not migrated away from groups and would appreciate having this in the export. 

So the follow-up ticket should be the way to go.